### PR TITLE
Adding dashboard view

### PR DESCRIPTION
**Pull request by katafira**
<h1>Purpose</h1>
Adding a dashboard view for the admin, so that he can see info about the system. Small changes in system administration view to separate it form this new one and avoid duplication.

Ready for revision.
PRs #37 and #38 should be merged before this one.
@sergio-ocon @delaluzparra @alexvkcr

### CODE REVIEW

**Comment by katafira** [comment]
@agarciavera @sebjimenez

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -13,13 +13,10 @@
 //= require jquery
 //= require jquery_ujs
 //= require turbolinks
-
-//
+//= require_tree .
```
Comment (sergio-ocon) [inline app/assets/javascripts/application.js]: Not needed (see other comments)

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -13,13 +13,10 @@
...
+//= require_self
 // Require elements for patternfly to work
 //
 //= require bootstrap
 //= require patternfly
-
-
-
-
+//= require bootstrap-switch
```
Comment (sergio-ocon) [inline app/assets/javascripts/application.js]: Yes, this is the only thing you need to do to add JS to the application

**Comment by katafira** [comment]
View already added in PR #37 and fixed.

**Pull request closed without merging**